src/parsing/location.ml

```diff
@@ -395,7 +395,7 @@ let () =
           Some (errorf ~loc:(in_file !input_name)
                 "I/O error: %s" msg)
 
-      | Misc.HookExnWrapper {error = e; hook_name;
+      (* | Misc.HookExnWrapper {error = e; hook_name;
```

Comment:
Do we want to leave a comment on why this is commented out?

Reply:
Done.